xbmc/cores/AudioEngine/Interfaces/AE.h

```diff
@@ -255,5 +255,7 @@ class IAE
    * Indicates if dsp addon system is active.
    */
   virtual bool HasDSP() { return false; };
+
+  virtual bool GetCurrentSinkFormat(AEAudioFormat &SinkFormat) { return false; }
```

Comment:
You're right I forgot the documentation ;)

Bool to show if all went well or if e.g. a new audio engine didn't support this feature to show that the request fails.